src/App.jsx

```diff
@@ -1,107 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentById(commentPostId) {
+  return commentsFromServer.find(comment => comment.postId === commentPostId)
+    || null;
+}
+
+export const posts = postsFromServer.map(post => ({
+  ...post,
+  user: getUserById(post.userId),
+  comments: getCommentById(post.commentPostId),
```

Comment:
```suggestion
  comments: getCommentById(post.id),
```
comments are never rendered because property commentPostId doesn't exist on post pls fix